### PR TITLE
Feedback on undo

### LINKED ISSUE #9192: Feedback on undo
When you press undo, show a small toast indicating what was undone, confirming that the undo order was sent.

Ideally, you should wait until undo is done succesfuly to show the message.

If you want to go the extra miles, and undo takes more than half a second to succeed, you can show a message confirming that undo was registered. This could occur if a long computation is being done in parallel, such as finding the next card in a big deck

### PR DESCRIPTION
**Pull request by Mohaned-Elfeky**
## Purpose / Description
Added feedback message(Toast) when an undo is done successfully.

## Fixes
Fixes #9192 

## Approach
added the toast message after the undo is done successfully and added the message in the dialogue resources.

## How Has This Been Tested?

Performed multiple undo's to check if the message is displayed or not.

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/res/menu/reviewer.xml [hunk 1]
```diff
@@ -119,8 +119,8 @@
     </item>
     <item
         android:id="@+id/action_search_dictionary"
-        android:title="@string/menu_select"
```
Comment (Arthur-Milchior) [inline AnkiDroid/src/main/res/menu/reviewer.xml:122]: Why ? I don't understand why you'd want a lookup. Can you please explain it in the commit message

**Comment by Arthur-Milchior** [comment]
Hi, 

Thanks a lot for this PR. AS a user, it will be really useful when I've a doubt about what I just undid (undone?)
Can you please rebase on top of master, as two files conflict.

Would you mind making a short video to show what you did. I'm confused by your code and fear that it was not what I expected. Might be good too, but I have trouble visualising what you did here

Since it's a UI change, I'd love you to use the accessibility scanner if you please. I realize that I'm not sure we ever tested it with toast, so it might be interesting. (see the link in the last checkbox).
Also, as second to last checkbox ask, I'd love a screenshot. It will be useful for translators, and also to see what it looks like.

File: AnkiDroid/src/main/res/values/03-dialogs.xml [hunk 2]
```diff
@@ -119,6 +119,7 @@
     <string name="dialog_positive_set">Set</string>
 
     <string name="lookup_button_content">Look up</string>
+    <string name="dialog_clipboard_empty">Clipboard is empty</string>
```
Comment (Arthur-Milchior) [inline AnkiDroid/src/main/res/values/03-dialogs.xml]: Why do you add anything about clipboard in an undo PR ? I'm totally lost

File: AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java [hunk 3]
```diff
@@ -1338,7 +1338,7 @@ protected void lookUpOrSelectText() {
             Timber.d("Clipboard has text = %b", clipboardHasText());
             lookUp();
         } else {
-            UIUtils.showThemedToast(AbstractFlashcardViewer.this, getApplicationContext().getString(R.string.dialog_clipboard_empty) , false);
+               selectAndCopyText();
```
Comment (Arthur-Milchior) [inline AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java]: Why do you remove and add this line?

**Review by Arthur-Milchior — COMMENTED** [review]
i'd like you to clear the Commit history if you can. I really am having trouble following the logic of your changes

File: AnkiDroid/src/main/res/menu/reviewer.xml (hunk 1 above)
Comment (Mohaned-Elfeky) [reply AnkiDroid/src/main/res/menu/reviewer.xml:122]: it is showing changes of a previous pull request for some reason

Comment (Akshay0701) [reply AnkiDroid/src/main/res/menu/reviewer.xml:122]: Maybe you created the current branch from Mohaned-Elfeky:fixes branch, this is the reason all previous commits is been showing in your current branch.

I will suggest creating a new branch from master and then do edits to it.

Comment (Mohaned-Elfeky) [reply AnkiDroid/src/main/res/menu/reviewer.xml:122]: Thanks I was planning on doing that

**Comment by Arthur-Milchior** [comment]
I feared you misunderstood the meaning of [rebase|https://git-scm.com/book/en/v2/Git-Branching-Rebasing], I understand the ambiguity, and am sorry for it.
If you're on our discord, don't hesitate to come we can discuss how to deal with git together. 
The other questions and remarks remain

**Pull request closed without merging**